bindings/ios/MEGASdk.mm

```diff
@@ -2604,9 +2605,13 @@ - (void)sendEvent:(NSInteger)eventType message:(NSString *)message {
 #pragma mark - Private methods
 
 - (MegaRequestListener *)createDelegateMEGARequestListener:(id<MEGARequestDelegate>)delegate singleListener:(BOOL)singleListener {
+    return [self createDelegateMEGARequestListener:delegate singleListener:singleListener queueType:ListenerQueueTypeMain];
```

Comment:
Use `ListenerQueueTypeMain` for backwards compatibility, so we don't need to change any existing code.

It is recommended to use `ListenerQueueTypeCurrent` if possible. I have audited CU, CU target folder and CU heartbeat related APIs to use `ListenerQueueTypeCurrent`.